### PR TITLE
adobe-air-sdk 18.0.0.143

### PR DESCRIPTION
**Pull request by vpmedia**
Updated Adobe AIR SDK to v18.0.0.143

Please merge it as soon as possible.

Thanks!

### CODE REVIEW

File: adobe-air-sdk-flex.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Find downloads at: https://helpx.adobe.com/air/kb/archived-air-sdk-version.html
 class AdobeAirSdkFlex < Formula
   homepage "https://www.adobe.com/devnet/air/air-sdk-download.html"
-  url "http://airdownload.adobe.com/air/mac/download/17.0/AdobeAIRSDK.tbz2"
-  sha256 "b13fcea757349692d82c14926d9afb1477387af0f3808f8ef7dbb7c373d3eb38"
-  version "17.0.0.144"
+  url "http://airdownload.adobe.com/air/mac/download/18.0/AdobeAIRSDK.tbz2"
+  sha256 "0e04a9ef57ed135e4b82e63fde91deee9e544836"
```
Comment (bfontaine) [inline adobe-air-sdk-flex.rb]: This looks like a SHA1.

File: adobe-air-sdk.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Find downloads at: https://helpx.adobe.com/air/kb/archived-air-sdk-version.html
 class AdobeAirSdk < Formula
   homepage "https://www.adobe.com/devnet/air/air-sdk-download.html"
-  url "http://airdownload.adobe.com/air/mac/download/17.0/AIRSDK_Compiler.tbz2"
-  sha256 "7fe1b5172fd4b2144751dace46b3335eededc28398ae4672db9fab4b1ab6c634"
-  version "17.0.0.144"
+  url "http://airdownload.adobe.com/air/mac/download/18.0/AIRSDK_Compiler.tbz2"
+  sha256 "d0cd59b779cf9b490406d72150cfd8fb34488c1e"
```
Comment (bfontaine) [inline adobe-air-sdk.rb]: Same here.

Comment (vpmedia) [inline adobe-air-sdk.rb]: Thanks for the reminder, I did miss this.

File: adobe-air-sdk-flex.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # Find downloads at: https://helpx.adobe.com/air/kb/archived-air-sdk-version.html
 class AdobeAirSdkFlex < Formula
   homepage "https://www.adobe.com/devnet/air/air-sdk-download.html"
-  url "http://airdownload.adobe.com/air/mac/download/17.0/AdobeAIRSDK.tbz2"
-  sha256 "b13fcea757349692d82c14926d9afb1477387af0f3808f8ef7dbb7c373d3eb38"
-  version "17.0.0.144"
+  url "http://airdownload.adobe.com/air/mac/download/18.0/AdobeAIRSDK.tbz2"
+  sha256 "5873bc33629e21cf9ebd6d9fa30f33813239374204bf4f1ab790c2e678bd9dfe"
+  version "18.0.0.144"
```
Comment (sugamasao) [inline adobe-air-sdk-flex.rb]: SDK for Mac version I think `18.0.0.143` is correct.
see: http://www.adobe.com/devnet/air/air-sdk-download.html
or
Try the Download SDK and `bin/adt -version`.

File: adobe-air-sdk.rb [hunk 4]
```diff
@@ -1,9 +1,9 @@
 # Find downloads at: https://helpx.adobe.com/air/kb/archived-air-sdk-version.html
 class AdobeAirSdk < Formula
   homepage "https://www.adobe.com/devnet/air/air-sdk-download.html"
-  url "http://airdownload.adobe.com/air/mac/download/17.0/AIRSDK_Compiler.tbz2"
-  sha256 "7fe1b5172fd4b2144751dace46b3335eededc28398ae4672db9fab4b1ab6c634"
-  version "17.0.0.144"
+  url "http://airdownload.adobe.com/air/mac/download/18.0/AIRSDK_Compiler.tbz2"
+  sha256 "36643ae1349ffde5e48a52ab37186460e1a5eb6051589e6e2004f6c1f865f3db"
+  version "18.0.0.144"
```
Comment (sugamasao) [inline adobe-air-sdk.rb]: Same here.

Comment (vpmedia) [inline adobe-air-sdk.rb]: Fixed - Thanks!

**Comment by vpmedia** [comment]
closing this one because it has been already updated.

**Pull request closed without merging**